### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by garrytrinder** [pnp/action-cli-login#25]
We cannot merge #24 until we update the @actions dependencies.

**Comment by milanholemans** [comment]
@garrytrinder I've added these fixes to the already existing PR. Do you want me to create a separate PR for it?

**Comment by garrytrinder** [comment]
Thanks for the update @milanholemans, I see you've updated the actions in the workflow.

Apologies, I should have been clearer, this issue relates to the @actions dependencies in `package.json`.

![image](https://github.com/pnp/action-cli-login/assets/11563347/52d513ee-06ee-453b-a062-212d9e97c20a)

**Comment by garrytrinder** [comment]
For example, when switching to `node@18` and running a build throws an error. 

![image](https://github.com/pnp/action-cli-login/assets/11563347/98d570e5-59df-42e3-a61e-fe4189c6a0f8)

**Comment by garrytrinder** [comment]
I didn't realise until I raised the PR that I was working from a very old `master` branch that was set as the default branch on my fork, however we had changed to use `main` branch about a year ago. 

It happily builds on `node@18`, however still worth updating the dependencies.

![image](https://github.com/pnp/action-cli-login/assets/11563347/6b605fa2-0dcb-43ed-bb72-3394ebf13850)

/cc @milanholemans
